src/components/dropbox-backup/DropboxBackup.jsx

```diff
@@ -0,0 +1,70 @@
+// @flow
+import React, {useState} from 'react';
+import {Dropbox} from 'dropbox';
+import './dropbox-backup.scss';
+
+type Props = {
+  notes: Array<Object>,
+};
+
+const getAccessToken = () => {
+  const accessToken = sessionStorage.getItem('dropbox_access_token');
+  if (accessToken) {
+    return accessToken;
+  }
+
+  const matches = window.location.hash.match(/access_token=(.+?)&/);
+  if (matches && matches[1]) {
+    sessionStorage.setItem('dropbox_access_token', matches[1]);
+    return matches[1];
+  }
+
+  return null;
+};
+
+const getAuthUrl = () => {
+  const dbx = new Dropbox({clientId: 'd41ntwo8ou6exgf', fetch});
```

Comment:
You will want to swap this out with your own clientId after creating a dropbox app. https://www.dropbox.com/developers/apps